maintainers/maintainer-list.nix

```diff
@@ -2862,6 +2862,12 @@
     githubId = 69209;
     name = "Ian Duncan";
   };
+  ianmjones = {
+    email = "ian@ianmjones.com";
+    github = "ianmjones";
+    githubId = 4710;
+    name = "Ian M. Jones";
+  };
```

Comment:
Could you add yourself to the mainatiner's list in a separate commit like?
```
maintainers: add ianmjones
```

Reply:
As in, just remove in one commit and then re-apply in separate commit to enable cherry picking?

Reply:
like the git log is
```
maintainers: add ianmjones
snippetpixie: init at 1.2.2
```

First commit touches `maintainer-list.nix` only.